projects/rero/ng-core/src/lib/record/record.ts

```diff
@@ -66,5 +66,15 @@ export interface SearchField {
 export interface SearchFilter {
   filter: string;
   label: string;
-  value?: string;
+  value: string;
+  /* If you set this value, the url parameter will still be present,
+  but with different values */
+  disabledValue?: string;
```

Comment:
Maybe we could use a boolean. If false, then the url parameter will be present but whitout any value (seems valid for URL's). Ex : http://localhost/?param1=stuff&param2